blurp-app/src/pages/blurpmap.jsx

```diff
@@ -167,10 +178,11 @@ const TestPage = () => {
                       </div>
                       <br />
                       <div>
+                        <label>Size</label>
                         <Slider
                           onChange={(e) => setSize(e.target.value)}
-                          min={20}
-                          max={120}
+                          min={1}

```

Comment:
Would 1 be too small or will the camera naturally look like a good size?

Reply:
It is a bit small, but the current camera ratio helps make it not appear too small, as the zoom affects how nodes and edges appear. It will probably be adjusted later as we decide what sizes would work better for the nodes/edges, and what camera ratio would work best.